src/common/hotbar-store.ts

```diff
@@ -94,7 +95,15 @@ export class HotbarStore extends BaseStore<HotbarStoreModel> {
 
   @action
   protected async fromStore(data: Partial<HotbarStoreModel> = {}) {
-    this.hotbars = data.hotbars;
+    if (data.hotbars?.length === 0) {
```

Comment:
Change this to 
```
if (!data.hotbars || !data.hotbars.length) {
```

original condition doesn't work when `hotbars` are `undefined`